apiv1/controllers/GoogleLoginController.js

```diff
@@ -75,5 +75,89 @@ module.exports = {
         }
 
         return resp.status(200).json(account);
+    },
+    getUserSetting : async function(req,resp){
+        let uid = req.headers['x-access-token'];
+        if(!uid){
+            return resp.status(401).json({
+                message: "No userID supplied",
+                error: "Unauthorized"
+            });
+        }
+
+        if(uid !== "1"){
+            uid = await verifyGoogleToken(req.headers['x-access-token']);
+            if(!uid){
+                return resp.status(401).json(invalid_token);
+            }
+        }
+
+        let setting; 
+        try {
+            const user = await GoogleLoginModel.findOne({_id:uid});
```

Comment:
Wouldn't this end up executing GoogleLoginModel.findOne({_id: '1'})?

Shouldn't this return a 404 in this instance?